client/src/services/getCurrentUser.js

```diff
@@ -1,4 +1,4 @@
-const getCurrentUser = () => {
+const getCurrentUser = async () => {
```

Comment:
we actually don't need to add an `async` here BECAUSE it's `return`ing a `fetch` request. `fetch` is, by nature, a Promise, so we can still `await` this successfully elsewhere even if it isn't labeled as `async`.

Reply:
corrected in next pull